package.json

```diff
@@ -69,6 +69,7 @@
     "annotations-oss": "oats ${REMOTE}contracts/priv/annotationd-oss.yml > ./src/client/annotationdRoutes.ts",
     "mapsd-oss": "touch ./src/client/mapsdRoutes.ts",
     "mapsd": "oats ${REMOTE}contracts/mapsd.yml > ./src/client/mapsdRoutes.ts",
+    "nifid": "oats ${REMOTE}contracts/priv/nifid.yml > ./src/client/nifidRoutes.ts",
```

Comment:
Regarding the lint-oss CI failure. Apparently the file _has_ to exist but we don't want the code to be there. So add a `"nifid-oss": "touch ./src/client/nifidroutes.ts"` target and include that in the generate-meta-oss step.

Also - can we rename `nifid` to `subscriptions` here? I think the pattern is to go by the feature name - mapsd is just not following that pattern.

Finally, the second lint-cloud job is failing because:
```
/home/circleci/influxdata/ui/src/writeData/subscriptions/components/SubscriptionsList.tsx
  4:1  error  Expected space or tab after '//' in comment  spaced-comment
```

Reply:
ok cool, updated to subscriptions and added that extra target

Reply:
resolved